ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query.hs

```diff
@@ -39,7 +39,9 @@ import qualified Data.Map.Strict as Map
 
 -- | Different queries supported by the ledger for all block types, indexed
 -- by the result type.
-data Query blk result = BlockQuery (BlockQuery blk result)
+newtype Query blk result = BlockQuery
+  { unQuery :: BlockQuery blk result
+  }
```

Comment:
Changing to a newtype is not ideal since this will soon be changed to

```haskell
data Query blk result where
    BlockQuery :: BlockQuert blk result -> Query blk result
    GetPartialLedgerConfig :: Query blk (PartialLedgerConfig blk)
```

In which case `unQuery` would be less useful. It seems you're not using the `unQuery` projection function anywhere. Do you plan on using it?

Reply:
I had expected to need it, but turns out not.  Will remove.